### PR TITLE
ss-14: Add Base Migrations

### PR DESCRIPTION
**Pull request by vasylpopovych**

### CODE REVIEW

**Review by nikita-remeslov — DISMISSED** [review]
Looks good to me
@VItaliy-Kalenichenko What are you think?

**Comment by VItaliy-Kalenichenko** [comment]
1. You should drop 'users' table and create it again with id of type uuid (use dropTableIfExists method)
2. I think it would be better to have all the migrations in one file, named InitialStructure

**Comment by vasylpopovych** [comment]
--

**Comment by nikita-remeslov** [comment]
> I added all migration to one file initial_structure. But have a problem. When try to run migration got the error. I think this error because something wrong in initial_structure file, but unfortunately I can't understand the reason.

Can you provide the error?

File: backend/src/migrations/20230215152530_add_users_table.ts
Comment (nikita-remeslov) [inline backend/src/migrations/20230215152530_add_users_table.ts:1]: We never change the existing migration

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Example 
image

Also, we should down all table in reverse order

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
Discussed in Slack

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**